### PR TITLE
Remove old Dashboard page

### PR DESCRIPTION
**Pull request by therealemjy**
Replace old dashboard page with the v2 one.

### CODE REVIEW

**Review by baomastr — APPROVED** [review]

**Comment by baomastr** [comment]
historical moment (:

File: src/pages/App.tsx [hunk 1]
```diff
@@ -14,7 +14,6 @@ import { Layout } from 'components';
 import { init as initTranslationLibrary } from 'translation';
```
Comment (coreyar) [inline src/pages/App.tsx:14]: We can `components/Dashboard`

**Comment by coreyar** [comment]
We can also remove `components/Dashboard`

**Comment by therealemjy** [comment]
> We can also remove `components/Dashboard`

Good catch, it's done.

**Pull request merged**